subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -2709,7 +2709,7 @@ static const uint8_t *adva_update(struct ll_adv_set *adv, struct pdu_adv *pdu)
 		if (0) {
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 		} else if (ll_adv_cmds_is_ext() && pdu->tx_addr) {
-			own_id_addr = ll_adv_aux_random_addr_get(adv, NULL);
+			own_id_addr = ull_adv_aux_random_addr_get(adv, NULL);
```

Comment:
hmmm, good point. This function tried to follow `ll_addr_get`, but looks like the memcpy part of the branch is unused in the source base. I will check uses of `ll_addr_get` too and make specific functions instead of the now redundant branching inside the functions.

Reply:
Are you going to do that in this PR?

Reply:
yes